ts/src/bybit.ts

```diff
@@ -1113,18 +1113,161 @@ export default class bybit extends Exchange {
                     'UNIFIED': 'unified',
                 },
                 'networks': {
+                    'BTC': 'BTC',
```

Comment:
technically we don't need to map the ones that are the same as the ccxt unified code 🤔

Reply:
yes yes right, but if it doesnt hurt let them keep it there for a while before I'll reach to base unification method updates, because iirc there were cases when we do smth like `if (networkCode in this.options['networks'])` or alike, so for a while let it be there.

after main base PR is done, it would handle everything accurately and we will remove them these odd entries

Reply:
maybe they are not really needed at this current moment, but I haven't looked into base methods yet and don't remind everything in details, but will do in 2 days or so, once I finish all individual exchange network mappings & network-id-code conversion updates.

Reply:
@ttodua ok we can keep them for now